Libraries/LibJS/Bytecode/Executable.h

```diff
@@ -47,7 +48,7 @@ struct SourceRecord {
     u32 source_end_offset {};
 };
 
-class Executable final : public Cell {
+class JS_API Executable final : public Cell {
```

Comment:
Classes like this immediately catch my eye. How is `JS::Bytecode::Executable` used outside of LibJS? I'm not seeing any users, but I could certainly be missing something.

Reply:
aha I see, thanks!

> Would you prefer if I go back through all the classes I annotated and explicitly annotate individual methods? 

I'm fine with exporting the class itself in these cases, since that's probably the simpler thing to do for now.